app/helpers/ingredients_helper.rb

```diff
@@ -2,28 +2,28 @@
 
 module IngredientsHelper
   def ingredient_display(ingredient)
-    display_name = [
-      qty_display(ingredient),
-      ingredient.measurement_unit,
-      ingredient.name,
-    ].join(' ')
-
-    if ingredient.preparation_style.present?
-      "#{display_name}: #{ingredient.preparation_style}"
+    display_name = if Ingredient::DESCRIPTIVE_UNITS.include?(ingredient.measurement_unit)
```

Comment:
Used in the Recipe show page to pluralize like:
* 6 whole carrots, chopped
* 3 cloves garlic, minced